src/libs/Network/index.ts

```diff
@@ -6,14 +6,24 @@ import pkg from '../../../package.json';
 import * as MainQueue from './MainQueue';
 import * as SequentialQueue from './SequentialQueue';
 
+let processQueueInterval: NodeJS.Timer;
+
 // We must wait until the ActiveClientManager is ready so that we ensure only the "leader" tab processes any persisted requests
 ActiveClientManager.isReady().then(() => {
     SequentialQueue.flush();
 
     // Start main queue and process once every n ms delay
-    setInterval(MainQueue.process, CONST.NETWORK.PROCESS_REQUEST_DELAY_MS);
+    processQueueInterval = setInterval(MainQueue.process, CONST.NETWORK.PROCESS_REQUEST_DELAY_MS);
 });
 
+// Clear interval
```

Comment:
NAB: This comment isn't really helpful. Can we please explain that this is needed only for tests?